src/signal_scope/signaldata.cpp

```diff
@@ -185,6 +185,13 @@ QRectF SignalData::computeBounds()
     if (y > maxY) maxY = y;
   }
 
+  double marginMax = 1.1;
+  double marginMin = 1.1;
+  if (maxY < 0.0) marginMax = 0.9;
+  if (minY > 0.0) marginMin = 0.9;
+  maxY *= marginMax;
+  minY *= marginMin;
```

Comment:
Yes, it is inflating the code. Somebody in the team wanted to inflate the bounds automatically. It should be removed.